sys/vane/jael.hoon

```diff
@@ -644,35 +647,103 @@
     ::    {$burn p/ship q/safe}
     ::
         $burn
-      %+  cure  our.tac
+      %^  cure  hen  our.tac
       abet:abet:(deal:(burb our.tac) p.tac [~ q.tac])
     ::
+    ::  boot from keys
+    ::    {$dawn =seed spon=(unit ship) czar=(map ship [=life =pass]) turf=(list turf)}
+    ::
+        %dawn
+      =*  our  who.seed.tac
+      ::  our initial private key, as a +tree of +rite
+      ::
+      =/  rit  (sy [%jewel (my [lyf.seed.tac key.seed.tac] ~)] ~)
+      =.  +>.$  $(tac [%mint our our rit])
+      ::  our initial galaxy table as a +map from +life to +public
+      ::
+      =/  kyz
+        %-  ~(run by czar.tac)
+        |=([=life =pass] `public`[live=| life (my [life pass] ~)])
+      =.  +>.$
+        %-  curd  =<  abet
+        (pubs:~(feel su hen our urb sub etn) kyz)
+      ::  XX sponsor
+      ::  XX reconcile with .dns.eth
+      ::  set initial domains
+      ::
+      =.  tuf.urb  turf.tac
+      ::
+      =.  moz
+        %+  weld  moz
+        ::  order is crucial!
+        ::
+        ::    %dill must init after %gall (and after %give intended for unix)
+        ::    %jael init must be deferred post %eyre init
```

Comment:
Brief reasoning here might be insightful.

Reply:
Abandon reason all ye who touch the boot sequence!